action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Image Actions'
+author: 'Calibre'
+description: 'Compresses Images for the Web'
+
+inputs:
+  githubToken:
+    description: 'GitHub Token'
+    required: true
+
+  jpegQuality:
+    description: 'JPEG quality level'
+    required: false
+    default: 80
+
+  pngQuality:
+    description: 'PNG quality level'
+    required: false
+    default: 80
+
+  webpQuality:
+    description: 'WEBP quality level'
+    required: false
+    default: 80
+
+  ignorePaths:
+    description: 'Paths to ignore during search'
+    required: false
+    default: "nodemodules/**"
+
+runs:
+  using: 'docker'
+  image: 'docker://calibreapp/github-image-actions' 
```

Comment:
Does this have any baring on how the action is executed? Does this make it possible to do what you've outlined in the README?:

```
- name: Compress Images
  uses: calibreapp/image-actions@master
```